xml/hwbp_registers.xml

```diff
@@ -270,23 +285,86 @@
             by 1.
         </field>
 
-        <field name="action" bits="5:0" access="R/W" reset="0">
-            Determines what happens when this trigger matches.
-
-            0: Raise a breakpoint exception. (Used when software wants to use the
-            trigger module without an external debugger attached.)
+        <field name="action" bits="5:0" access="R/W" reset="0" />
```

Comment:
Again I think you still need to point to the Action table inside the description here.

Reply:
Done.